src/components/Register.jsx

```diff
@@ -53,6 +62,43 @@ const Register = ({ toggleForm }) => {
       return;
     }
 
+    const hasUpperCase = /[A-Z]/.test(values.password);
```

Comment:
Can you also think of a good place in the UI to specify that a special character and capital letter are required without needing to press submit?